modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -5,20 +5,339 @@ endif::[]
++
+image::os-as-http-listener-config.png[]
++
+. In the connector's main screen, set the Path to the `/store` value.
++
+image::os-as-http-path.png[]
++
+. Click ObjectStore in the Mule Palette and drag the *Store* operation to the canvas.
+. Specify the Key as `#[payload.key]` and the Value with the `payload.value` string.
+. Click the green plus sign to the right of the Object Store field, and in the Global
+Element Properties screen, click OK to accept the defaults.
+. Click the HTTP Listener again and drag it to the canvas under the first flow to create a second
+flow. Use the same settings as in Step 7. In the main screen, set the Path to the
+`retrieve` value.
++
+image::os-as-2nd-flow-http.png[]
++
+. Click ObjectStore in the Mule Palette and drag the *Retrieve* operation to the canvas.
+. In the Key field, specify the `#[attributes.queryParams.key]` value.
+. Click the green plus sign to the right of the Object Store field, and in the Global
+Element Properties screen, click OK to accept the defaults.
+. You can change the flow labels by double-clicking the flow labels. Change the first flow to
+`Store` and the second flow to `Retrieve`.
++
+When done, the flows should appear as:
++
+image::os-as-finished-flows.png[]
+
+You now have a completed Studio application. You can test that it runs correctly by 
+clicking Run As > Mule Application. This is just a test. Once you get your application
+to run, you can stop it.
+
+=== Test Your Studio 7 Application
+
+. In Studio, right click your project in the Package Explorer.
+. Click *Anypoint Platform* > *Deploy to CloudHub*. Note: CloudHub is name of the server, but
+you access your application in Anypoint Platform > Runtime Manager.
++
+image::os-as-deploy-to-cloudhub.png[]
++
+. After your application appears in Anypoint Platform > Runtime Manager, start it so that
+the application is running. Note the `cloudhub.io` URL.
+. From the command line, enter this curl command:
++
+[source,console]
+----
+$ curl -X POST -H "Content-Type: application/json" -d '{ "key": "MyKey", "value": "OSv2 Test" }' "http://YOUR_CH_URL/store"
+----
++
+Replace YOUR_CH_URL with the `cloudhub.io` URL for your application in Runtime Manager.
++
+You can improve readability for this command if you append `;echo` to the end of the command.
++
+. In Runtime Manager, click *CloudHub* in the Server column for your application to display details, and click *Manage Application*.
++
+image::os-rm-manage-app.png[]
++
+. Click *Object Store* and click the Object Store, then the Partition, and the Key. You can
+see the value as `[binary value] BINARY`. 
++
+You can see that `MyKey` made it into the Object Store user interface, which confirms how to
+view that keys successfully can be sent through your Design Center application and stored in 
+the Object Store.
++
+. To view the value for the key, use this curl command:
++
+[source,console]
+----
+curl http://osv2-qoav.us-w1.cloudhub.io/retrieve?key=MyKey
```

Comment:
And again here:
Doesn't this link need to change based on the customer's application name?  Shouldn't we use the same approach as was used above where we make the user copy in the link from design center as LINK_FROM_DC in the url?

Reply:
Fixed as requested.